### PR TITLE
HONKS token

### PR DESCRIPTION
**Pull request by CollinEnstad**
HONK HONK

### CODE REVIEW

**Review by p2pcash — CHANGES_REQUESTED** [review]
It needs a comma here after the } .

Also you should try to squash your commits if you can.

File: manifest.json [hunk 1]
```diff
@@ -23,4 +23,11 @@
     "tokenId":
       "fc7aa9a70b4ae86d30b93a98e853716a4217e77c60769f664da3c6189647d3ba"
   }
+  {
+    "symbol": "HONKS",
+    "name": "HONKS",
+    "tokenId":
+      "f35007140e40c4b6ce4ecc9ad166101ad94562b3e4f650a30de10b8a80c0b987"
+  },
```
Comment (p2pcash) [inline manifest.json]: You should remove this trailing comma too.

File: manifest.json [hunk 2]
```diff
@@ -23,4 +23,11 @@
     "tokenId":
       "fc7aa9a70b4ae86d30b93a98e853716a4217e77c60769f664da3c6189647d3ba"
   }
```
Comment (p2pcash) [inline manifest.json]: It needs a comma after the }

**Comment by p2pcash** [comment]
Could you also squash your commits into a single one ?

**Comment by CollinEnstad** [comment]
fixed, but i have 0 idea how to squash, google doesnt help. sorry, github noob.

**Comment by cgcardona** [comment]
This has been handled manually.

**Pull request closed without merging**